functions-and-operators/string-functions.md

```diff
@@ -163,11 +163,76 @@ Insert a substring at the specified position up to the specified number of chara
 
 ### [`INSTR()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_instr)
 
-Return the index of the first occurrence of substring.
+The `INSTR(str, substr)` function is used to get the index of the first occurrence of the second given argument `substr` 
```

Comment:
Please put lines 167 to 169 all on one line.  Or put a empty line in between them if that makes more sense. Or change them to be a list.